src/EnergyPlus/api/state.cc

```diff
@@ -70,6 +70,7 @@ void stateReset(EnergyPlusState state)
 {
     auto *this_state = reinterpret_cast<EnergyPlus::EnergyPlusData *>(state);
     this_state->clear_state();
+    this_state->init_constant_state(*this_state);
```

Comment:
Thanks. I think this is a lot cleaner. Sorry for not putting down more documentation about this new pattern to begin with.